pixart.js

```diff
@@ -1 +1,36 @@
+var button = document.querySelector("#set-color");
+var colorText = document.querySelector("#color-field");
+var bigSquare = document.querySelector(".brush");
+var color = "#1B4370";
+var pixels = document.querySelector(".pixels");
 
+playPixart();
+
+function playPixart() {
+  button.addEventListener("click", setColor); //This also works when "enter" is pressed on the keyboard. Why? Should I create a separate event to ensure this event happens for keying enter on any computer?
+  addSquares();
+  pixels.addEventListener("mouseover", clickSquare);
+}
+
+//When I click the "Set Color" button, it should change the color of the "brush" box to the color I specify in the input field.
+function setColor(event) {
+  event.preventDefault();
+  color = colorText.value;
+  bigSquare.style.backgroundColor = color;
+}
+
+//Create 8000 divs of the "square" class and append them to the body
+function addSquares() {
+  for (var i = 0; i < 8000; i++) {
+    var newDiv = document.createElement("div");
```

Comment:
Nice job on this for loop :+1:  I really like how you broke down each step and abstracted `document.createElement("div")` into `var newDiv`